app/scripts/storage/storage-file-cache.js

```diff
@@ -18,18 +18,17 @@ const StorageFileCache = StorageBase.extend({
         if (this.path) {
             return callback && callback();
         }
-        try {
-            const path = Launcher.getUserDataPath('OfflineFiles');
-            const fs = Launcher.req('fs');
-            if (!fs.existsSync(path)) {
-                fs.mkdirSync(path);
+
+        const path = Launcher.getUserDataPath('OfflineFiles');
+
+        Launcher.fileExists(path, exists => {
+            if (!exists) {
```

Comment:
Please always maintain callback consistency.
Not doing so will result in weird and hard to debug app hangs.
❗️ Added missing callbacks.